app/build.gradle

```diff
@@ -1,5 +1,12 @@
 apply plugin: 'com.android.application'
 
+allprojects {
+    repositories {
+        jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
Acording to [that this](https://forum.f-droid.org/t/maven-google-com/519) maven.google.com seems not to be supported by fdroid.